lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -8,147 +11,121 @@
 
 # NOTE: TkAgg tests seem to have interactions between tests,
 # So isolate each test in a subprocess. See GH#18261
+# Each of the tests decorated with _tk_test must be fully self-contained, and
+# thus perform all the imports it needs.
+
+
+def _tk_test(count, func=None):
+    """
+    Run *func* in a subprocess, and assert it prints "success" *count* times.
+    """
```

Comment:
```suggestion
def _isolated_tk_test(count, func=None):
    """
    A decorator to run *func* in a subprocess, and assert it prints "success"
    *success_count* times.

    TkAgg tests seem to have interactions between tests, so isolate each test
    in a subprocess. See GH#18261
    
    The decorated function must be fully self-contained, and thus perform
    all the imports it needs.
    """
```

And delete the comments above.

Reply:
yup